packages/public/server/__tests-e2e__/entities/staticPage.ts

```diff
@@ -0,0 +1,78 @@
+/**
```

Comment:
For the Future: Different tasks should be solved in different PRs -> so I can already merge your first PR while you still work on the second.